tests/release/pipelines/fbc_release.go

```diff
@@ -310,6 +361,18 @@ func createFBCReleasePlanAdmission(fbcRPAName string, managedFw framework.Framew
 	})
 	Expect(err).NotTo(HaveOccurred())
 
+	var structData map[string]interface{}
```

Comment:
wouldn't be easier just define a string var and add it to the existent `fbc` struct? For instace:

```
var targetIndex string
	if (isStagedIndex) == "true" {
	
        } else {
                  targetIndex = constants.TargetIndex
	}
```
Then assign the `targetIndex` directly in the  data struct above ?

Reply:
You are right. But I want to confirm if we need to test there is no "targetIndex" field in the data when the ""stagedIndex" is "true".

Reply:
got it. Resolving it then! thanks!